rules/rules.libsonnet

```diff
@@ -72,7 +72,7 @@
             // it is used to calculate per-node metrics, given namespace & instance.
             record: 'node_namespace_instance:kube_pod_info:',
```

Comment:
strictly the name of this recording rule should change too?  at least make it clearer by saying `node_namespace_podname` or something.

Reply:
Agreed, in that case we should probably default to “pod” instead of “instance” as well. Wdyt?